example/lambda.json

```diff
@@ -11,5 +11,13 @@
     "/*.pyc"
   ],
   "timeout": 30,
-  "memory": 512
+  "memory": 512,
+  "vpc": {
```

Comment:
@jarosser06 I _think_ it's more future-friendly to just test against `/test/configs` .. configs, so that's how I set it up now. Please take another look and let me know if you agree.